examples/imagenet/imagenet_main.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""ImageNet example.
+
+This script trains a ResNet-50 on the ImageNet dataset.
+The data is loaded using tensorflow_datasets.
+"""
+
+from absl import app
+from absl import flags
+
+import tensorflow.compat.v2 as tf
```

Comment:
/tensorflow.compat.v2/tensorflow

Reply:
Again outdated file :) 

Replaced it in all instances of imagenet example.